src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -154,8 +154,7 @@ public void verifyTopLeaderboard() {
   }
 
   public void verifyNoAdsOnPage() {
-    scrollToSelector(AdsContent.getSlotSelector(AdsContent.ADS_IN_CONTENT_CONTAINER));
-    scrollToSelector(AdsContent.getSlotSelector(AdsContent.PREFOOTERS_CONTAINER));
+    scrollToSelector(AdsContent.getSlotSelector(AdsContent.BOTTOM_LB));
```

Comment:
Shouldn't be a part of logic in verifyNoAd?

Can we reuse SLOT_TRIGGERS in verifyNoAd and scroll to given slot when it's necessary?

https://github.com/Wikia/selenium-tests/blob/7e8de247fd9bc0d96769993f1ca7794218cd6086/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L96